File: java/core-fw-spring/src/main/java/com/firelord/spring/component/rpc/http/vo/RespErrVo.java
package com.firelord.spring.component.rpc.http.vo;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import java.util.ArrayList;
import java.util.List;

/**
 * RespErrVo
 */
@ToString
public class RespErrVo {
    //#region Fields

    /**
     * error items
     */
    @Setter
    @Getter
    private List<RespErrItemVo> items = new ArrayList<>();

    //#endregion
}

File: cs/core-fw-spring/component/rpc/http/vo/RespErrVo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace core_fw_spring.component.rpc.http.vo
{
    public class RespErrVo
    {
        #region Fields

        private List<RespErrItemVo> items = new List<RespErrItemVo>();

        public List<RespErrItemVo> Items { get => items; set => items = value; }

        #endregion
    }
}
